FantasmoSDK/app/src/main/java/com/fantasmo/sdk/filters/FMFrameFilterChain.kt

```diff
@@ -98,6 +101,7 @@ class FMFrameFilterChain(context: Context) {
     @RequiresApi(Build.VERSION_CODES.JELLY_BEAN_MR1)
     fun accepts(fmFrame: FMFrame): FMFrameFilterResult {
         if (shouldForceAccept()) {
+            imageEnhancer?.accepts(fmFrame)
```

Comment:
Just double checking that this really will enhance the image? I get it operates on the `fmFrame` but I don't see an assignment, so I guess my question is does the fmFrame variable act as a pointer or reference (if I have my computer science terms correct, which I may not)

Reply:
It gets passed by reference, and the method changes the image in place inside of the frame. That's exactly what happens in the other place in the code where the enhancement happens. Closing the PR.